skeleton/config/services.yml.twig

```diff
@@ -51,7 +51,7 @@ services:
         arguments:
             - '@config'
         calls:
-            - [set_name, [{{ EXTENSION.vendor_name }}.{{ EXTENSION.extension_name }}.cron.task.{{ EXTENSION.extension_name|lower }}]]
+            - [set_name, [{{ EXTENSION.vendor_name }}.{{ EXTENSION.extension_name }}.cron.task.sample]]
```

Comment:
Just noticesd that the 3 file names changed also appear in the skeleton.yml and need to be changed there as well.